Dockerfile.glibc.profiling

```diff
@@ -0,0 +1,66 @@
+#base stage
+FROM archlinux:base-devel AS builder
+
+# Force-sync packages, install archlinux-keyring, repopulate keys
+RUN pacman -Syy
+RUN pacman -S archlinux-keyring --noconfirm --disable-download-timeout
+RUN rm -rf /etc/pacman.d/gnupg/* && pacman-key --init && pacman-key --populate archlinux
+
+# Install updates
+RUN pacman -Syu --noconfirm --disable-download-timeout
+
+# Set up Rust toolchain
+RUN pacman -S rustup mold wget --noconfirm --disable-download-timeout
+RUN rustup toolchain install nightly-2024-01-12
+RUN rustup default nightly-2024-01-12
+RUN rustup target add wasm32-unknown-unknown
+RUN rustup component add rust-src
+
+# builder stage
+WORKDIR /iroha
+COPY . .
+RUN mold --run cargo build --target x86_64-unknown-linux-gnu --features wasm_profiling --profile profiling 
```

Comment:
I assume profile is the only difference from `Dockerfile.glibc`?

Reply:
Yes

Reply:
how about using [build arguments](https://docs.docker.com/build/guide/build-args/)?